packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -1968,6 +1968,11 @@ class _CupertinoTimerPickerState extends State<CupertinoTimerPicker> {
 
   // The currently selected values of the picker.
   int? selectedHour;
+
+  FixedExtentScrollController? _hourScrollController;
+  FixedExtentScrollController? _minuteScrollController;
+  FixedExtentScrollController? _secondScrollController;
```

Comment:
Can you put all controllers after all selected... members, for better readability?

Reply:
Tests in G3 passed, so it is only thing remaining.